packages/delivery-service/src/validation/notification/enableOrDisableChannelValidation.ts

```diff
@@ -0,0 +1,42 @@
+import { validateNotificationChannelType } from './notificationChannelValidation';
+
+// validates enable/disable notification channel data
+export const validateToggleNotificationChannel = (
+    notificationChannelType: string,
+    isEnabled: boolean,
+): {
+    isValid: boolean;
+    errorMessage?: string;
+} => {
+    try {
+        // validate notification channel type
+        const isChannelValid = validateNotificationChannelType(
+            notificationChannelType,
+        );
+
+        // return error if channel type is not valid
+        if (!isChannelValid.isValid) {
+            return isChannelValid;
+        }
+
+        // throw error if isEnabled not found
+        if (isEnabled !== false && !isEnabled) {
```

Comment:
what is this ?

Reply:
this statement never will be true

Reply:
I don't know that this does and have found it [in the backend,](https://github.com/dm3-org/dm3/blob/b97020fb0befefee621e00c0547d7f401f6f6864/packages/backend/src/validation/notification/enableOrDisableChannelValidation.ts#L22) too.

Reply:
removed from backend cfc0dfa

Reply:
I can make this statement true manually in node:
```
> enOrDis.validateToggleNotificationChannel("EMAIL",null);
{ isValid: false, errorMessage: 'isEnabled value is missing' }
```